src/components/TodoApp/TodoApp.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import cn from 'classnames';
+
+import { TodoList } from '../TodoList';
+import { getTodos } from '../../api/todos';
+import { useTodosContext } from '../store';
+import { useUncomplitedTodos } from '../../helpers/useUncomplitedTodos';
+import { TodoFilter } from '../TodoFilter';
+import { useComplitedTodos } from '../../helpers/useComplitedTodos';
+
+const USER_ID = 12027;
+
+export const TodoApp: React.FC = () => {
+  const { todos, reciveTodos } = useTodosContext();
+
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState('');
+
+  const compTodosLength = useComplitedTodos().length;
```

Comment:
Do not abbreviate names to unreadable ones
```suggestion
  const completedTodosLength = useCompletedTodos().length;
```